tests/integration/test_wazuh_db/test_wazuhdb_getconfig.py

```diff
@@ -0,0 +1,101 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: integration
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks the usage of the wazuhdb getconfig command used to get the current configuration
+tier: 0
+modules:
+    - wazuh_db
+components:
+    - manager
+daemons:
+    - wazuh-db
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+tags:
+    - wazuh_db
+'''
+import os
+import pytest
+import yaml
+from wazuh_testing.constants.paths import WAZUH_PATH
+from wazuh_testing.utils.database import query_wdb
+from wazuh_testing.utils.file import get_list_of_content_yml
+
```

Comment:
Missing line break, a line break could be added to preserve the spacing of the rest of the file.